External link CSS-customization for mobile-layout (#590)
jellyfin/jellyfin-docs: general/clients/css-customization.md

![Screenshot of watched indicators with a custom transparent color applied](~/images/custom-css-transparentwatched.png)

### Display external links in mobile layout

The mobile app disables display of external links to IMDb, TheMovieDB, Trakt, etc by default.

> To enable the external links so they are display add the following:

Reviewer: ```suggestion
To enable the external links again, add the following snippet:
```

Author: updated as per suggestion.
